sdk/src/rent.rs

```diff
@@ -54,7 +59,9 @@ impl Rent {
             (0, true)
         } else {
             (
-                ((self.lamports_per_byte_year * data_len as u64) as f64 * years_elapsed) as u64,
+                (((ACCOUNT_STORAGE_OVERHEAD * self.lamports_per_byte_year)
```

Comment:
nice, but what if you de-distributed this?

```
self.lamports_per_byte_year * (ACCOUNT_STORAGE_OVERHEAD + data_len as u64)

Reply:
done